src/saving.jl

```diff
@@ -25,6 +25,10 @@ function Base.show(io::IO, saved_values::SavedValues)
                 "\nt:\n", saved_values.t, "\nsaveval:\n", saved_values.saveval)
 end
 
+@recipe function plot(saved_values::SavedValues)
+    (saved_values.t, saved_values.saveval)
```

Comment:
Plots.jl won't plot a `Vector{Vector}` correctly like this. It'll plot each `t` with each `saveval[i]`. If you just make this return `DiffEqArray(saved_values.t, saved_values.saveval)` then it'll do the proper transform for Plots.jl, which requires a transpose and thus requires the indexing. See https://github.com/JuliaDiffEq/RecursiveArrayTools.jl/blob/master/src/vector_of_array.jl#L92.